scripts/bootstrap/src/metadata.script.ts

```
@@ -0,0 +1,169 @@
+
+    const metadataRepository = new KyselyMetadataCache(db, schema);
+
+    const publicGatewayProvider = new CachingMetadataProvider(
+        new PublicGatewayProvider(PUBLIC_GATEWAY_URLS, logger),
+        metadataRepository,
+        logger,
+    );
+
+    // const envioIndexerClient = new EnvioIndexerClient(INDEXER_URL, INDEXER_SECRET);
+    const cids: string[] = [];
+    let hasMoreEvents = true; // Flag to control the loop
```

ESLint

[error] 'hasMoreEvents' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)